pages/feed/index.tsx

```diff
@@ -222,10 +223,12 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
             <FollowingHighlightWrapper selectedFilter={selectedRepo} emojis={emojis} />
           </TabsContent>
         </Tabs>
-        <div className="hidden mt-10 md:block">
+        <div className="hidden gap-6 mt-10 w-[18.625rem] md:flex md:flex-col">
```

Comment:
```suggestion
        <div className="hidden gap-6 mt-10 w-72 md:flex md:flex-col">
```
w-[18.625rem] this is not scalable, w-72 => 18rem